### PR TITLE
Fix: Fixed an issue where the properties window had the wrong drag region

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Related: #10044
- Related: #9919

**Comments**
@yair100 now `DragRegion` stretches for all the title bar except over `NavigationView`

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by yaichenbaum** [comment]
Looks like it has the same issue.

**Comment by ferrariofilippo** [comment]
Are you using mouse or touch?

**Comment by ferrariofilippo** [comment]
https://user-images.githubusercontent.com/102259289/193466209-888e6e8f-e470-478b-9dbb-bc43223839b2.mp4

**Comment by yaichenbaum** [comment]
Mouse

**Comment by ferrariofilippo** [comment]
I searched online to see if this is a known bug and indeed it is (<a href="https://github.com/microsoft/WindowsAppSDK/issues/2953">GitHub issue</a>)
They say it's a WinApp SDK bug, fixed in 1.2 preview (which I'm currently using).
I believe I should change the code since `SetDragRectangles` is not compatible with win-10, what do you think?

**Comment by ferrariofilippo** [comment]
This should solve the problem, let me know it that works

File: src/Files.App/Views/Pages/Properties.xaml.cs [hunk 1]
```diff
@@ -117,9 +117,13 @@ private void NavigationView_SizeChanged(object? sender, SizeChangedEventArgs? e)
...
 				// This area is over the top margin of NavigationView.
-				new RectInt32(0, 0, navigationViewWidth, (int)NavigationView.ActualOffset.Y),
+				new RectInt32(0, 0, (int)(navigationViewWidth + NavigationView.ActualOffset.X), (int)NavigationView.ActualOffset.Y),
+				// This area is over the left margin of NavigationView.
+				new RectInt32(0, (int)NavigationView.ActualOffset.Y, (int)NavigationView.ActualOffset.X, (int)NavigationView.ActualSize.Y),
+				// This area is over the bottom margin of NavigationView.
+				new RectInt32(0, (int)(NavigationView.ActualSize.Y + NavigationView.ActualOffset.Y), (int)(navigationViewWidth + NavigationView.ActualOffset.X), (int)NavigationView.Margin.Bottom),
 				// This area is on the right of NavigationView and stretches for all the remaining space.
-				new RectInt32(navigationViewWidth, 0, (int)(TitleBarDragArea.ActualSize.X - navigationViewWidth), (int)TitleBarDragArea.ActualSize.Y)
+				new RectInt32(navigationViewWidth, 0, (int)(TitleBarDragArea.ActualSize.X - navigationViewWidth), (int)NavigationView.DesiredSize.Height)
```
Comment (yaichenbaum) [inline src/Files.App/Views/Pages/Properties.xaml.cs:126]: Are these changes still needed?

**Comment by yaichenbaum** [comment]
The issue is still there :thinking

**Comment by ferrariofilippo** [comment]
Are you working with WinApp SDK 1.1?

**Comment by yaichenbaum** [comment]
We're currently using the latest experimental build.

**Comment by ferrariofilippo** [comment]
I searched online if this is a known bug and indeed it is <a href="https://github.com/microsoft/WindowsAppSDK/issues/2953">GitHub Issue</a>.
I'll try to switch to `Window.SetTitleBar`, as said in the issue.

**Comment by ferrariofilippo** [comment]
> I'll try to switch to `Window.SetTitleBar`, as said in the issue.

It seems we can't go this way, because in `WinUI3` the content set using `Window.SetTitleBar` doesn't receive user input.
Due to this, the issue will remain in Win10 until WinAppSDK is extended.

**Comment by yaichenbaum** [comment]
I'm having this issue in Windows 11.

**Comment by ferrariofilippo** [comment]
> I'm having this issue in Windows 11.

It was to let you know that this will still be a bug in Win10, no matter if I can fix this in Win11

**Comment by yaichenbaum** [comment]
I have a workaround we can implement for Windows 10 but we need to fix Windows 11 first.

**Comment by ferrariofilippo** [comment]
I just tested it in the preview and it works fine (except for areas that are added with this PR)

**Comment by yaichenbaum** [comment]
It's still not working on build 25227

**Comment by ferrariofilippo** [comment]
@yair100 I still can't reproduce, can you mark this as `Help Wanted`

**Comment by yaira2** [comment]
This was resolved with #10571
